draft-ietf-taps-interface.md

```diff
@@ -2332,27 +2332,59 @@ a data transfer useful. It is given in bits per second. The special value -1 ind
 that no bound is specified.
 
 
-### TCP-specific Property: User Timeout {#tcp-uto}
+## TCP-specific Properties: User Timeout Option (UTO) {#tcp-uto}
```

Comment:
This section defines only one TCP-specific property, so plural is very confusing / misleading here.

Reply:
Done. However, there are multiple configuration parameters...